polaris/dataset/converters/_sdf.py

```diff
@@ -141,6 +141,7 @@ def _get_name(mol: dm.Mol):
         annotations = {self.mol_column: ColumnAnnotation(is_pointer=True, modality=Modality.MOLECULE_3D)}
         if self.smiles_column is not None:
             annotations[self.smiles_column] = ColumnAnnotation(modality=Modality.MOLECULE)
+            annotations[self.smiles_column] = ColumnAnnotation(content_type=ContentType.SMILES)
```

Comment:
**Suggestion**
I think the second line might be overwriting the first line here. This might be better:
```suggestion
            annotations[self.smiles_column] = ColumnAnnotation(modality=Modality.MOLECULE, content_type=ContentType.SMILES)
```